test/expected/hnsw_dist_func.out

```diff
@@ -0,0 +1,224 @@
+) v ORDER BY v.dist, v.id;
+INFO:  began scanning with 0 keys and 1 orderbys
+INFO:  starting scan with dimensions=3 M=16 efConstruction=128 ef=64
+INFO:  usearch index initialized
+ id  | dist 
+-----+------
+ 010 | 0.00
+ 000 | 1.00
+ 011 | 1.00
+ 110 | 1.00
+ 001 | 2.00
+ 100 | 2.00
+ 111 | 2.00
+(7 rows)
+
+-- this should use index
+EXPLAIN SELECT * FROM (
+    SELECT id, ROUND(l2sq_dist(vector, '[0,1,0]'::vector)::numeric, 2) as dist
+    FROM small_world_l2_vec
+    ORDER BY vector <-> '[0,1,0]'::vector LIMIT 7
+) v ORDER BY v.dist, v.id;
+                                                          QUERY PLAN                                                           
+-------------------------------------------------------------------------------------------------------------------------------
+ Sort  (cost=0.72..0.74 rows=7 width=48)
+   Sort Key: v.dist, v.id
+   ->  Subquery Scan on v  (cost=0.00..0.62 rows=7 width=48)
+         ->  Limit  (cost=0.00..0.55 rows=7 width=56)
+               ->  Index Scan using small_world_l2_vec_vector_idx on small_world_l2_vec  (cost=0.00..84.10 rows=1070 width=56)
+                     Order By: (vector <-> '[0,1,0]'::vector)
+(6 rows)
+
+-- cos
+SELECT * FROM (
+    SELECT id, ROUND(cos_dist(vector, array[0,1,0])::numeric, 2) as dist
+    FROM small_world_cos
+    ORDER BY vector <-> array[0,1,0] LIMIT 7
+) v ORDER BY v.dist, v.id;
+INFO:  began scanning with 0 keys and 1 orderbys
+INFO:  starting scan with dimensions=3 M=16 efConstruction=128 ef=64
+INFO:  usearch index initialized
+ id  | dist 
+-----+------
+ 010 | 0.00
+ 011 | 0.29
+ 110 | 0.29
+ 111 | 0.42
+ 001 | 1.00
+ 100 | 1.00
+ 101 | 1.00
+(7 rows)
+
+-- this should use index
+EXPLAIN SELECT * FROM (
+    SELECT id, ROUND(cos_dist(vector, array[0,1,0])::numeric, 2) as dist
+    FROM small_world_cos
+    ORDER BY vector <-> array[0,1,0] LIMIT 7
+) v ORDER BY v.dist, v.id;
+                                                       QUERY PLAN                                                        
+-------------------------------------------------------------------------------------------------------------------------
+ Sort  (cost=0.70..0.72 rows=7 width=48)
+   Sort Key: v.dist, v.id
+   ->  Subquery Scan on v  (cost=0.00..0.60 rows=7 width=48)
+         ->  Limit  (cost=0.00..0.53 rows=7 width=52)
+               ->  Index Scan using small_world_cos_vector_idx on small_world_cos  (cost=0.00..81.42 rows=1070 width=52)
+                     Order By: (vector <-> '{0,1,0}'::real[])
+(6 rows)
+
+-- ham
+SELECT * FROM (
+    SELECT id, ROUND(hamming_dist(vector, array[0,1,0])::numeric, 2) as dist
+    FROM small_world_ham
+    ORDER BY vector <-> array[0,1,0] LIMIT 7
+) v ORDER BY v.dist, v.id;
+INFO:  began scanning with 0 keys and 1 orderbys
+INFO:  starting scan with dimensions=3 M=16 efConstruction=128 ef=64
+INFO:  usearch index initialized
+ id  | dist  
+-----+-------
+ 010 |  0.00
+ 011 |  7.00
```

Comment:
How is hamming distance calculated here?
I would expect the distance to be 1.00 not 7.00